api/src/main/java/uk/gov/hmcts/PaymentApiApplication.java

```diff
@@ -22,15 +21,14 @@
 import org.springframework.plugin.core.OrderAwarePluginRegistry;
 import org.springframework.plugin.core.PluginRegistry;
 
-import javax.servlet.ServletContextListener;
+import jakarta.servlet.ServletContextListener;
 import java.util.Arrays;
 import java.util.concurrent.TimeUnit;
 
 @EnableCaching
 @EnableFeignClients
 @EnableAsync
 @SpringBootApplication
-@EnableCircuitBreaker
```

Comment:
Updated from Hystrix to Resilience4j. Resilience4j doesn't require @EnableCircuitBreaker as it integrates with Spring Boot automatically.